src/route/page.rs

```diff
@@ -118,8 +118,8 @@ pub async fn get_deepwell_page_wrapped(
     deepwell: web::Data<DeepwellPool>,
 ) -> HttpResponse {
     match get_deepwell_page(wiki_id, slug, &deepwell).await {
-        Ok(o) => o,
-        Err(()) => get_deepwell_page(wiki_id, "_404", &deepwell).await.unwrap(), // todo: make this safer
+        Some(o) => o,
+        None => get_deepwell_page(wiki_id, "_404", &deepwell).await.unwrap(), // todo: make this safer
```

Comment:
In cases where you have 
```rust
match item {
    Some(x) => x,
     None => something_else(n),
}
```
you can do
```
item.unwrap_or_else(|| something_else(n));
```
instead

(or
```
item.unwrap_or(10)
```
for constants or cheap evaluations)

Reply:
This pattern doesn't seem to work, since the `None` pattern here requires an async evaluation.

Reply:
mhmm that is true. we should just work on getting the page handlers set up so we can call that instead.